libraries/types/types.eos

```diff
@@ -71,6 +72,7 @@ struct transfer
    from      AccountName  # may not be the message.sender if message.sender has delegated authority by from
    to        AccountName
    amount    UInt64
+   memo      String
```

Comment:
Seems like we should limit the size of the string. Currently, this is a std::string.